example-config/readme.md

```diff
@@ -0,0 +1,21 @@
+# Running `etesync` under `nginx` and `uwsgi`
+
+This configuration assumes that etesync server has been installed in the home folder of a non privileged user 
+called `EtesyncUser` following the instructions in <https://github.com/etesync/server-skeleton>. Also that static 
+files have been collected by running the following command from within the `etesync` folder.
+
+    ./manage.py collectstatic
+
+Before the above command, it might be necessary to add the line 
+
+    STATIC_ROOT = os.path.join(BASE_DIR, 'static/')
```

Comment:
It's actually easier to have collect static elsewhere, so for example in `/srv/http/etesync_server` or something because you then don't need to have nginx have permissions to the etesync source dir.
If you don't put it elsewhere you'd have to `o+x` (xx1) permission on all the directories leading to the source which is possibly undesirable.
What I do is have it stored in /srv/http/etesync_server and have a symlink from /path/to/etesync_server/static to that directory.

Does that make sense?

Also, I just added this line to the default config, but please the symlink comment I made here.